crm_deduplicate_by_website/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============================
+Deduplicate Contacts by website
+===============================
+
+This module extends the criteria to match duplicated contacts using the field
+website.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to *Sales > Tools > Deduplicate Contacts*.
+#. Mark "Website" in the section "Search duplicates based on duplicated data
+   in".
+#. This criteria will be used for deduplicating.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/111/9.0
```

Comment:
this is wrong but if you move to new readme structure we won't care about it anymore :wink: 
https://github.com/OCA/maintainer-tools/tree/master/template/module/readme